build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Z tego co pamiętam to rozwiązaliśmy to inaczej, przez zmianę LF CRLF. To można pominąć